### ISSUE TITLE
No run_all.fish

### ISSUE THREAD

**Issue by Anaphory** [evolaemp/online_cognacy_ident#6]
The README talks about some fish-shell script
https://github.com/evolaemp/online_cognacy_ident/blob/3b998aeaee3e0058da5d758cefb54bc2e37a2e94/README.md#L100
which does not exist. I have taken the steps to convert `scripts/reproduce.fish` into `zsh` (which is mostly the cryptic line

 	if [[ ${ipa_datasets[(ie)$dataset]} -le ${#ipa_datasets} ]]

instead of the nice

	if contains $dataset $ipa_datasets

but at least that works).

However, I noticed that that file uses the same training data `training_data/asjpv17_word_pairs.txt` on all datasets. Is that the intended use case?

**Comment by PhyloStar** [comment]
You are right. We dont have to use the same training data for all datasets. The algorithm should work for any dataset unsupervisedly.

**Comment by Anaphory** [comment]
Cool! I have it running, just fiddling with it to get it to play nicely with CLDF and non-ASJP sound classes. I'll tell you what my results are when I have them!
